### ISSUE TITLE
Networkx not working after updating nbconvert (export to PDF)

### ISSUE THREAD

**Issue by rafmora** [networkx/networkx#4616]
I needed to export a jupyter notebook to PDF and required the installation xelatex. I installed MikTex and conda update nbconvert. Then, I uninstalled MikTex, but networkx is still not showing the graph.

### Current Behavior
The graphs, that were working well before the update, did not show on Jupyter Notebook anymore. However, there is no error produced while running the code
### Expected Behavior
Produce the graph

### Steps to Reproduce
fontMulti=1.0
def cause_effect_plot(dataset): #Directed Acyclic Graphs
    nrel_node= dict(dataset.degree)
    nrel_node_max=nrel_node[max(nrel_node.keys(), key=(lambda k: nrel_node[k]))]
    nrel_node_min=nrel_node[min(nrel_node.keys(), key=(lambda k: nrel_node[k]))]
    
    nodes_edges(dataset)
    rel=dataset.edges.data('weight')
 
    weight_linethickness = [(i * 3) for i in list(nx.get_edge_attributes(dataset, 'weight').values())]
 
    pos= nx.spring_layout(dataset)
    options = {"with_labels": False, "node_color": "white", "edgecolors": "blue"}
    nx.draw_networkx(dataset, pos=pos, node_size = [nrel_node[k]*170 for k in nrel_node], width = weight_linethickness, **options)    #, pos=bayes_pos,   
    
    fontm=(24-8)/(nrel_node_max-nrel_node_min+1)
    for node, (x, y) in pos.items():
        text(x, y, node, fontsize=((nrel_node[node]/nrel_node_max)*fontm+10)*fontMulti) #, ha='center', va='center'
    fig.show()
    return

edges1 = [('health', 'absences'),('health','Grade_H1')]
student1 = StructureModel(edges1)
student1['health']['absences']['weight']=0.1
student1['health']['Grade_H1']['weight']=0.2

fig, ax = plt.subplots(figsize=(12,5)) 
cause_effect_plot(student1) # print weight_linethickness = 2 * weight

### Environment

Python version: Python 3.7.9
NetworkX version: 2.5
jupyter core     : 4.7.1
jupyter-notebook : 6.2.0
qtconsole        : 5.0.2
ipython          : 7.20.0
ipykernel        : 5.3.4
jupyter client   : 6.1.7
jupyter lab      : 3.0.1
nbconvert        : 5.5.0
ipywidgets       : 7.6.3
nbformat         : 4.4.0
traitlets        : 5.0.5

**Comment by rossbar** [comment]
I'm not sure I understand what you mean by "produce the graph" and I'm having trouble parsing the provided example, partly because the formatting seems to be off and there's no indication of where the error is actually occurring. An updated, minimal reproducing example would help track down what the problem might be.

Absent that, since this seems related to uninstalling/reinstalling packages aside from NetworkX, my initial suspicion would be that it's likely an installation conflict or environment management problem. There's no reason that updating `nbconvert` should have any direct impact on networkx, so it's likely either that:
 1. The environment with the updated nbconvert is not the same as the environment with the working NetworkX version, or
 2. There's something wrong with the Jupyter/nbconvert/latex installation.

**Comment by rafmora** [comment]
Thanks Ross et. Al. for your response.
Last night, after a long try & error exercise  I found that Pandas & Pandas-Profiling newer versions disable networkx to display the Directed Acyclic Graph.I downgraded to pandas==0.25.3 and pandas-profiling==2.4.0 and networkx==2.5 worked.
Thank You

Sent from Yahoo Mail on Android 
 
  On Mon, Feb 15, 2021 at 3:31 AM, Ross Barnowski<notifications@github.com> wrote:   


I'm not sure I understand what you mean by "produce the graph" and I'm having trouble parsing the provided example, partly because the formatting seems to be off and there's no indication of where the error is actually occurring. An updated, minimal reproducing example would help track down what the problem might be.

Absent that, since this seems related to uninstalling/reinstalling packages aside from NetworkX, my initial suspicion would be that it's likely an installation conflict or environment management problem. There's no reason that updating nbconvert should have any direct impact on networkx, so it's likely either that:
   
   - The environment with the updated nbconvert is not the same as the environment with the working NetworkX version, or
   - There's something wrong with the Jupyter/nbconvert/latex installation.
